src/lib/TestUtils.ts

```diff
@@ -0,0 +1,37 @@
+import I18n, { Api } from './I18n'
+
+const mockNavigation: any = {
+  navigate: jest.fn(),
+  addListener: jest.fn(() => jest.fn()),
+  replace: jest.fn(),
+  goBack: jest.fn(),
+  popToTop: jest.fn(),
+  canGoBack: jest.fn(),
+  setOptions: jest.fn(),
+  getState: jest.fn(() => ({ index: 0 })),
+}
+
+const mockNavigationContext: any = {
+  isFocused: jest.fn(() => true),
+  addListener: jest.fn(() => jest.fn()),
+  navigate: jest.fn(),
+}
+
+function getTestI18n(api: Api = { setLanguage: () => {} }): I18n {
```

Comment:
Deze functie is nieuw en maakt test vertalingen aan: een object waarbij elke key die wordt bevraagd zijn eigen naam returnt:
```
{
  key1: "key1",
  "key twee", "key twee",
}
```
zonder dat je alle keys hoeft aan te maken. Een beetje TypeScript magic.